LeakyJeansOPE/xCode9DatabaseExample/GameScene.swift

```diff
@@ -16,25 +16,31 @@ class GameScene: SKScene {
     
     override func didMove(to view: SKView) {
         
-        // Get label node from scene and store it for use later
-        self.label = self.childNode(withName: "//helloLabel") as? SKLabelNode
-        if let label = self.label {
-            label.alpha = 0.0
-            label.run(SKAction.fadeIn(withDuration: 2.0))
-        }
+        //Create the background object and give it to the SceneGraph-- Change size to be drawn in display
+        let background = SKSpriteNode(imageNamed: "MainMenu.png")
+        addChild(background)
+        background.size = CGSize(width: size.width, height: (size.height / 2) - 250)
+        //background.position = CGPoint(x: size.width / 2, y: size.height / 2)
         
-        // Create shape node to use during mouse interaction
-        let w = (self.size.width + self.size.height) * 0.05
-        self.spinnyNode = SKShapeNode.init(rectOf: CGSize.init(width: w, height: w), cornerRadius: w * 0.3)
-        
-        if let spinnyNode = self.spinnyNode {
-            spinnyNode.lineWidth = 2.5
-            
-            spinnyNode.run(SKAction.repeatForever(SKAction.rotate(byAngle: CGFloat(Double.pi), duration: 1)))
-            spinnyNode.run(SKAction.sequence([SKAction.wait(forDuration: 0.5),
-                                              SKAction.fadeOut(withDuration: 0.5),
-                                              SKAction.removeFromParent()]))
-        }
+//        // Get label node from scene and store it for use later
```

Comment:
try not to commit commented out code, think of it like a book, you are only committing final draft material.